tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -237,14 +240,20 @@ install_cc_image() {
 	local component="rootfs-image"
 	local root_hash_vanilla="root_hash_vanilla.txt"
 	local root_hash_tdx=""
+	local kernel_version=""
 	if [ -n "${tee}" ]; then
+		jenkins="${jenkins_url}/job/kata-containers-2.0-rootfs-image-${tee}-cc-$(uname -m)/${cached_artifacts_path}"
 		if [ "${tee}" == "tdx" ]; then
-			jenkins="${jenkins_url}/job/kata-containers-2.0-rootfs-image-${tee}-cc-$(uname -m)/${cached_artifacts_path}"
 			component="${tee}-rootfs-image"
 			root_hash_vanilla=""
 			root_hash_tdx="root_hash_${tee}.txt"
 		fi
-	fi
+		if [ "${tee}" == "sev" ]; then
+			component="${tee}-rootfs-initrd"
+			root_hash_vanilla=""
+			kernel_version="-$(get_kernel_image_name)"
```

Comment:
Mind to explain why the kernel version is the kernel image name?

Reply:
That variable name should have been `kernel_last_commit` instead of `kernel_version` because it basically tracks that. I will push changes to rename that.